.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: python
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
+    - env: TOXENV=flake8
```

Comment:
This to be reads like we run flake8 under 3.8, but tox.ini says 3.6. Can we pick one?